nntrainer/layers/lstm.cpp

```diff
@@ -52,6 +60,11 @@ void LSTMLayer::finalize(InitLayerContext &context) {
   output_dim = input_dim;
   output_dim.width(unit);
 
+  if (dropout_rate > epsilon) {
+    wt_idx[LSTMParams::dropout_mask] = context.requestTensor(
+      output_dim, "LSTM:dropout_mask", true, ITERATION_LIFESPAN);
```

Comment:
`true` ->  `false`
https://github.com/nnstreamer/nntrainer/pull/1427#discussion_r676238140

Reply:
I will fix it.